Kitodo/src/main/java/org/kitodo/services/image/ImageService.java

```diff
@@ -100,4 +103,20 @@ Image getScaledWebImage(URI imageFileUri, double percent) throws IOException {
     Image getSizedWebImage(URI imageFileUri, int pixelWidth) throws IOException {
         return imageManagement.getSizedWebImage(imageFileUri, pixelWidth);
     }
+
+    /**
+     * Return singleton variable of type FilterService.
+     *
+     * @return unique instance of FilterService
+     */
+    public static ImageService getInstance() {
+        if (Objects.equals(instance, null)) {
+            synchronized (FilterService.class) {
```

Comment:
Why are you synchronizing on `FilterService.class` and not on  `ImageService.class` as you are inside ImageService class?

Reply:
Changed